airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/JdbcSqlOperations.java

```diff
@@ -43,12 +44,27 @@ protected JdbcSqlOperations(final DataAdapter dataAdapter) {
 
   @Override
   public void createSchemaIfNotExists(final JdbcDatabase database, final String schemaName) throws Exception {
-    if (!schemaSet.contains(schemaName) && !isSchemaExists(database, schemaName)) {
-      database.execute(String.format("CREATE SCHEMA IF NOT EXISTS %s;", schemaName));
-      schemaSet.add(schemaName);
+    try {
+      if (!schemaSet.contains(schemaName) && !isSchemaExists(database, schemaName)) {
+        database.execute(String.format("CREATE SCHEMA IF NOT EXISTS %s;", schemaName));
+        schemaSet.add(schemaName);
+      }
+    } catch (Exception e) {
```

Comment:
maybe we should log the original exception here, since it's hard to guarantee that subclasses always handle it correctly? I'm always a bit conflicted about this, since we might end up logging the same exception multiple times

Reply:
Since the use case hasn't presented itself I'm going to err on the side of inaction and not assume anything just yet. As for now the current behavior will log the original exception and the snowflake subclass will pass along the original exception in the Config Exception